docs/TerraToolkitUpgradeGuide-v5.0.0.md

```diff
@@ -0,0 +1,116 @@
+# Terra Toolkit Upgrade Guide v5.0.0
+This document will provide information on upgrading from terra-toolkit 4.x to 5.0.0.
+
+## Terra Scripts
+### Serve
+Serve is now a thin abstraction on webpack dev server and the commandline api is now identical. With this addition it now means that you have control over the dev server through options specified in your webpack config as well as through the cli.
+
+Why use serve instead of webpack-dev-server directly? Having the serve abstraction provides a hook for us to change the servers implementation in case webpack-dev-server nolonger meets our needs.
+
+Our default webpack config has been updated with the following defaults:
+
+```javascript
+module.exports = {
+  devServer: {
+    // We set the host this way to allow it to be served from docker containers.
+    host: '0.0.0.0',
+    // These options cut down on noise in the webpack build output.
+    stats: {
+      colors: true,
+      children: false,
+    },
+  },
+};
+```
+
+Serve is nolonger available as a javascript function. If this is needed use webpack-dev-server directly.
+
+### Serve Static
+Webpack-dev-server now supports IE 10+, because of this change we have removed the ability for serve-static to run webpack to create your site. Serve static now will simply host static site content.
+
+Serve static will now return 404.html (if it exists) if an html page is not found.
+
+These api options have been removed from both the cli and javascript:
+* config
+* production
+* disk
+
+If you want to serve a non hot-reloading site without pre-building your site, use tt-serve-static with the following options specified in your webpack config.
```

Comment:
Should this be use **tt-serve-static~ **tt-serve** to disabled hot relaoding in the the webpack-dev-server?